### PR TITLE
Working with API

### PR DESCRIPTION
**Pull request by IbrohimRasulov**
✅  Create a new game with the name of your choice by using the API.
✅  Implement the "Refresh" button (receiving data from the API and parsing the JSON).
✅  Implement the form "Submit" button (sending data to the API).
✅  Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/Game.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const GAME_ID = 'EtQZjbZctTfC60wlEG9f';
```
Comment (SajjadAhmad14) [inline src/Game.js:1]: Please configure code linter from [this](https://github.com/microverseinc/linters-config/tree/master/javascript) config and make sure all the checks are passing on PR.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ibrohim, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git Flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/Game.js (hunk 1 above)
Comment (IbrohimRasulov) [reply src/Game.js:1]: I have added linter correctly, but I don't know why they are not working

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @IbrohimRasulov, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**